spec/classes/init_spec.rb

```diff
@@ -8,22 +8,19 @@
         let(:facts) {{
           :osfamily => osfamily,
         }}
-
-        it { should include_class('make::params') }
-
-        it { should contain_class('make::install') }
       end
     end
   end
 
-  context 'unsupported operating system' do
-    describe 'make class without any parameters on Solaris/Nexenta' do
+  context '"unsupported" operating system' do
```

Comment:
The code no longer checks for supported OS's and fails by default, so this check for unsupported OS should be removed entirely.